samples/Common.c

```diff
@@ -1,6 +1,9 @@
 #define LOG_CLASS "WebRtcSamples"
 #include "Samples.h"
 
+#define KVS_DEFAULT_MEDIA_SENDER_THREAD_STACK_SIZE 64 * 1024
```

Comment:
Can we add units to the name of the variable (or maybe a comment indicating the units). Same to the readme

Reply:
Perhaps also, we can leave a comment indicating how this was determined.